lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
```

Comment:
I think where this shows up for the end user is currently only the dropdown menu in the editor, where unavailable markups will be red (or grey or some such) and unselectable. I actually think it might be a nice idea to have something like a gollum info or settings page where this kind of stuff is also displayed.